loadtest.py

```diff
@@ -28,17 +28,31 @@ def _memoize(*args, **kwargs):
 
 
 @memoize
-def get_payload_and_headers(size, compressed=False):
-    """Returns a payload and headers
+def get_payload_and_headers(size, dump_names=None, compressed=False):
+    """Returns a payload and headers ready for posting
 
     :arg size: the size in bytes the payload should be
+    :arg dump_names: list of dumps to include; default is
+        ``['upload_file_minidump']``
     :arg compressed: whether or not to compress the payload
 
     :returns: ``(payload, headers)``
 
     """
+    if not dump_names:
+        dump_names = ['upload_file_minidump']
+
     # Generate the payload and headers for a crash
-    raw_crash, dumps = utils.generate_sized_crashes(size, compressed)
+    raw_crash, dumps = utils.generate_sized_crashes(
+        size, dump_names=dump_names, compressed=compressed
+    )
+
+    # Make sure we have the specified dumps
+    assert list(sorted(dumps.keys())) == list(sorted(dump_names)), (
+        'Dumps are missing. There is a bug in the test code. %s vs. %s' % (
+            dumps.keys(), dump_names
+        )
+    )
```

Comment:
If we add a bug to the code such that it's not producing all the dumps, this will fail.